.github/workflows/pr_checks.yml

```diff
@@ -3,26 +3,36 @@ on:
   pull_request:
 
 jobs:
-  backend-unit-tests:
-    name: Backend Unit Tests
+  tests:
+    name: Containerized Tests
     runs-on: self-hosted
     env:
-      BUILD_DIR: tmp/pr_builds/PR_${{ github.event.number }}/backend_unit_tests
+      BUILD_DIR: tmp/pr_builds/PR_${{ github.event.number }}/
       DOCKER_PROJECT_NAME: backend-unit-tests-${{ github.event.number }}-${{ github.run_id }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
             path: ${{ env.BUILD_DIR }}
 
-      - name: Run Backend Unit Tests with Docker
+      - name: Backend Unit Tests with Docker
         working-directory: ${{ env.BUILD_DIR }}
-        run: docker compose -p ${{ env.DOCKER_PROJECT_NAME }} -f compose.testing.yml run --build backend-unit-tests
+        run: make backend_tests PROJECT_NAME=${{ env.DOCKER_PROJECT_NAME }}
```

Comment:
Replaced with make command.

Might be nice for output sake to do build in a separate step so only test output is shown under this step, but I didn't feel like making another permutation of make commands.